db/migrate/20190104181652_convert_all_responses_to_use_english_pdf.rb

```diff
@@ -0,0 +1,13 @@
+class ConvertAllResponsesToUseEnglishPdf < ActiveRecord::Migration[5.2]
+  class Response < ::ActiveRecord::Base
+    self.table_name = :responses
+  end
+
+  def up
+    Response.update_all pdf_template_reference: 'et3-v1-en'
```

Comment:
This will ensure that all old records are stated to have used et3-v1-en template, so if we ever have to re generate pdf's, we can do